rs/nns/governance/src/storage/neurons.rs

```diff
@@ -44,9 +44,51 @@ pub(crate) struct StableNeuronStoreBuilder<Memory> {
     pub transfer: Memory,
 }
 
+trait Finite {
+    const MIN: Self;
+    const MAX: Self;
+}
+
+// Implementations of Finite. (If Rust were like Go, these identical
+// implementations would not be necessary.)
+
+impl Finite for u64 {
+    const MIN: Self = Self::MIN;
```

Comment:
If these are already implemented, is it because the keys implement Storable?  Could you use that instead of a new interface?